lib/QubitInformationObjectXmlCache.class.php

```diff
@@ -81,12 +81,17 @@ public function export($resource)
    *
    * @return null
    */
-  public function exportAll()
+  public function exportAll($options = array())
   {
+    $skip = isset($options['skip']) ? $options['skip'] : 0;
```

Comment:
Just doing that to keep the class self contained (avoiding an "Undefined index" notice in case anything else ends up calling the method without providing that in the $options array).